lang/en/contact.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "title" => "Write",
+    "title2" => "to Us!",
+    "company" => "Blumilk Sp. z o.o.",
+    "subtitle" => "Are you interested enough to want to discuss your project with us, exchange ideas, become our business partner, or work with us? We would love to talk with you!",
+    "form.email" => "E-mail",
```

Comment:
```suggestion
    "form" => [
        "email" => "E-mail",
    ],
```

Would be good to change it here and in other places also